TF2 Rich Presence/settings.py

```diff
@@ -28,7 +31,7 @@ def access_registry(save: Optional[dict] = None) -> Optional[dict]:
 
     try:
         reg_key_data: dict = json.loads(winreg.QueryValue(reg_key, 'Settings'))
-    except FileNotFoundError:  # means that the key hasn't been initialized
+    except (FileNotFoundError, json.JSONDecodeError):  # means that the key hasn't been initialized
```

Comment:
I could not find what is suppose to throw FileNotFoundError so I simply fell back to catching JSONDecodeError

Reply:
Raised by winreg.QueryValue() I believe

Reply:
Checked this on Windows and it does indeed throw FileNotFound.

Reverted this change and implemented throwing this exception in the compatibility library